ipfx/bin/plot_ephys_nwb.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import matplotlib.pyplot as plt
+import numpy as np
+from ipfx.data_set_utils import create_data_set
+
+DEFAULT_NWB_FILE_NAME ="/allen/programs/celltypes/workgroups/279/MIES Testing Spring 2018/Sst-IRES-CreAi14-395722.01.01.01.nwb"
+
+
+def plot_data_set(data_set, sweep_table, nwb_file_name):
+
+    height_ratios, width_ratios = axes_ratios(sweep_table)
+
+    fig, ax = plt.subplots(len(height_ratios), 3,
+                           figsize=(15, 7),
+                           gridspec_kw={'height_ratios': height_ratios, 'width_ratios': width_ratios}
+                           )
+
+    i_ax = 0
+    for stimulus_code, sweep_set_table in sweep_table.groupby("stimulus_code"):
```

Comment:
Can't you use `enumerate` (https://docs.python.org/2/library/functions.html#enumerate) here?

Reply:
Agree, definitely use enumerate over manually incrementing `i_ax`.